.env.example

```
@@ -22,6 +22,15 @@ POSTGRES_PASSWORD=
 POSTGRES_DB=aurora_db
 POSTGRES_HOST=postgres
 POSTGRES_PORT=5432
+DB_POOL_MIN=2
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_POOL_MIN key should go before the POSTGRES_DB key

(UnorderedKey)